src/seedu/addressbook/data/person/Address.java

```diff
@@ -38,19 +49,23 @@ public static boolean isValidAddress(String test) {
 
     @Override
     public String toString() {
-        return value;
+    	String Address = valueBlock.toString() + ADDRESS_CONCATENATION_SYMBLE + 
+    			valueStreet.toString() + ADDRESS_CONCATENATION_SYMBLE +    			
+    			valueUnit.toString() + ADDRESS_CONCATENATION_SYMBLE + 
+    			valuePostalCode.toString();
```

Comment:
For multi line declarations, try to start a new line with the operation symbol instead of leaving it at the back.